incident-response/analysis.md

```diff
@@ -0,0 +1,8 @@
+# [Incident Response](./README.md) / Incident Analysis and Retrospectives
+
+Once an incident has ended, we want to perform an analysis and conduct a
```

Comment:
> Once an incident has ended, we want to perform an analysis and conduct a retrospective to figure out what we can learn from what happened.

I think it's important to call out a blameless retrospective here. From a psychological perspective, I think the wording brings more participation.

Reply:
Yes, that will definitely be a part of the actual section when I work on it. Right now this is just a placeholder for something that will be done for https://www.pivotaltracker.com/story/show/172926162.

Reply:
Added a some blameless verbiage there.